projects/1.16/assets/tinkers-construct/tconstruct/book/mighty_smelting/zh_cn/defense/protection/fire.json

```
@@ -0,0 +1,14 @@
+{
+  "modifier_id": "tconstruct:fire_protection",
+  "text": [
+    {
+      "text": "穿过烈火与烈焰。\nThrough the fire and the flames."
```

```suggestion
      "text": "穿过烈焰与火光。\nThrough the fire and the flames."
```
在网上找了歌词。原来文本里“烈”字重复了。